suitcase/utils/conftest.py

```diff
@@ -0,0 +1,111 @@
+import bluesky # noqa
+from bluesky.tests.conftest import RE # noqa
+from bluesky.plans import count # noqa
+import event_model # noqa
+from ophyd.tests.conftest import hw # noqa
+import pytest # noqa
+from . import UnknownEventType # noqa
+
+hw = hw()
+
+
+# Some useful plans for use in testing
+def simple_plan(dets):
+    '''A simple plane which runs count with num=5'''
+    yield from count(dets, num=5)
+
+
+def multi_stream_one_descriptor_plan(dets):
+    '''A plan that has two streams but on descriptor per stream)'''
+    @bluesky.preprocessors.baseline_decorator(dets)
+    def _plan(dets):
+        yield from count(dets)
+
+    yield from _plan(dets)
+
+
+@pytest.fixture(params=[[hw.det], [hw.direct_img], [hw.det, hw.direct_img]],
```

Comment:
If the items in `params` must be strings, you could use `[['det'], ...]` and then use `getattr` in the function body.

Reply:
thanks have implemented this behaviour